src/Abc.Zebus/ZebusLogManager.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Abc.Zebus
+{
+    public static class ZebusLogManager
+    {
+        private static ILoggerFactory _loggerFactory = NullLoggerFactory.Instance;
+
+        public static ILoggerFactory LoggerFactory
```

Comment:
**question:** Why did you choose to expose `ILoggerFactory` and not `ILoggerProvider`?

Reply:
`ILoggerFactory` seemed to be the standard approach, and it's [conceptually more high-level](https://blog.stephencleary.com/2018/06/microsoft-extensions-logging-part-2-types.html).